src/MAUI/Maui.Samples/Samples/Data/EditFeatureAttachments/EditFeatureAttachments.xaml.cs

```diff
@@ -27,6 +27,7 @@ namespace ArcGIS.Samples.EditFeatureAttachments
         tags: new[] { "JPEG", "PDF", "PNG", "TXT", "data", "image", "picture" })]
     public partial class EditFeatureAttachments : ContentPage
     {
+#pragma warning disable CA1422
```

Comment:
Why are you ignoring obsolete warnings for this entire class? 1. shouldn't it just be foe the specific call that causes the warning, and 2, shouldn't we avoid obsolete code in our samples?

Reply:
Looks like [UIImagePickerController](https://developer.apple.com/documentation/UIKit/UIImagePickerController) needs to be replaced with [PHPickerViewController](https://developer.apple.com/documentation/photosui/phpickerviewcontroller).

also

`UIApplication.SharedApplication.KeyWindow` is obsolete after iOS 13.0. 

Probably worth updating both of these and removing the pragma warning.

Reply:
@dotMorten removing obsolete warning for entire class and isolating it to one call.

@duffh I was able to resolve the `UIApplication.SharedApplication.KeyWindow` warning. I wasn't planning on replacing the UIImagePickerController on this PR. I found several bugs in the EditFeatureAttachments sample which might require a larger diff and more extensive testing. I did open an issue for this. 

Thanks both for good feedback.